### ISSUE TITLE
Hard state changes when it should not

### ISSUE THREAD

**Issue by gjongenelen** [Icinga/icingadb#156]
Hi, I've been testing icingadb for a while now, and it works great. But I have one concern:

Why is icingadb reporting a hard-state change when only the soft-state changed once (1/3)? 
<img width="498" alt="Screenshot 2020-04-03 at 22 18 54" src="https://user-images.githubusercontent.com/5789343/78401651-83659300-75f9-11ea-8b7d-77fe685a42cf.png">

In my opinion this should be a soft-state change twice.

Version: rc1

**Comment by Al2Klimov** [comment]
@nilmerg What's you opinion?

**Comment by nilmerg** [comment]
Well, OK/UP seem to be always hard. That's the same for the IDO.

**Comment by Al2Klimov** [comment]
I.e. it works as intended.

**Comment by lippserd** [comment]
@nilmerg @flourish86 We could write "Host/service recovered" maybe even with "... from (hard/soft) state" here instead of just "hard state changed", which would make it more obvious. What do you think?

**Comment by nilmerg** [comment]
Good idea. Although *Recovered* should suffice. We don't write *Host hard state changed* either.
